tidb-cloud/import-csv-files.md

```diff
@@ -74,77 +74,77 @@ Because CSV files do not contain schema information, before importing data from
 
-        - **Object Name Pattern**: enter a pattern that matches the names of the CSV files to be imported. If you have one CSV file only, you can enter the filename here directly.
+        - **File name**: enter a pattern that matches the names of the CSV files to be imported. If you have one CSV file only, enter the file name here directly.
 
             For example:
 
             - `my-data?.csv`: all CSV files starting with `my-data` and one character (such as `my-data1.csv` and `my-data2.csv`) will be imported into the same target table.
             - `my-data*.csv`: all CSV files starting with `my-data` will be imported into the same target table.
 
-        - **Target Table Name**: enter the name of the target table in TiDB Cloud, which must be in the `${db_name}.${table_name}` format. For example, `mydb.mytable`. Note that this field only accepts one specific table name, so wildcards are not supported.
+        - **Target table**: enter the name of the target table in TiDB Cloud, which must be in the `${db_name}.${table_name}` format. For example, `mydb.mytable`. Note that this field only accepts one specific table name, so wildcards are not supported.
 
-4. Click **Import**.
+    - **Table Filter**: If you want to filter which tables to be imported, you can specify one or more table filters in this field, separated by `,`.
 
-    A warning message about the database resource consumption is displayed.
+        For example:
+
+        - `db01.*`: all tables in the `db01` database will be imported.
+        - `db01.table01*,db01.table02*`: all tables starting with `table01` and `table02` in the `db01` database will be imported.
+        - `!db02.*`: except the tables in the `db02` database, all other tables will be imported. `!` is used to exclude tables that do not need to be imported.
+        - `*.*` : all tables will be imported.
+
+        For more information, see [table filter snytax](/table-filter.md#syntax).
+
+5. Click **Next**.
+
+6. On the **Preview** page, check and update the CSV specific configurations, including separator, delimiter, header, not-null, null, backslash-escape, and trim-last-separator. You can click the **Click here to edit csv configuration** link in the upper-right corner of the preview table to change the CSV configurations.
+
+    > **Note:**
+    >
+    > For the configurations of separator, delimiter, and null, you can use both alphanumeric characters and certain special characters. The supported special characters include `\t`, `\b`, `\n`, `\r`, `\f`, and `\u0001`.
 
-5. Click **Confirm**.
+7. Click **Start Import**.
 
     TiDB Cloud starts validating whether it can access your data in the specified bucket URL. After the validation is completed and successful, the import task starts automatically. If you get the `AccessDenied` error, see [Troubleshoot Access Denied Errors during Data Import from S3](/tidb-cloud/troubleshoot-import-access-denied-error.md).
```

Comment:
"the import task starts automatically" .Not start automatically, just go to step 2

Reply:
Yes corrected already. See this revision https://github.com/pingcap/docs/pull/10265/files#r966552945